src/app_feup/lib/model/app_state.dart

```diff
@@ -34,7 +35,9 @@ class AppState {
       'printBalanceStatus': RequestStatus.none,
       'feesStatus': RequestStatus.none,
       'coursesStateStatus': RequestStatus.none,
-      'lastUserInfoUpdateTime': null
+      'lastUserInfoUpdateTime': null,
+      'books': <Book>[],
```

Comment:
```suggestion
      'searchBooks': <Book>[],
```
In order to be consistent with the other PRs